tools/proofers/PPage.inc

```diff
@@ -87,6 +87,12 @@ class PPage
         return "$projects_url/{$this->lpage->projectid}/{$this->lpage->imagefile}";
     }
 
+    function url_for_display_image()
```

Comment:
For consistency, this should take a `$escape_amp` argument and do the escaping (or not) like the other functions.